### DISCUSSION TITLE
CC3517DF-FB34-457D-A479-B36EA5FEEF9C

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#28] [General]
# Rhea's rings

Rhea, Saturn's second largest moon, may have a possible very faint ring system consisting of three dense narrow bands within a disk of particles. This would be the first discovery of rings around a moon. The potential discovery was announced on March 6, 2008 in the journal Science, based on analyzes made with data from the Cassini spacecraft. However, the existence of these possible rings has not been fully confirmed.
NOTE: The rings are very, very thin. So, in Celestia, you'll need to get very close to Rhea in order to see them.

https://celestia.mobi/addon?item=CC3517DF-FB34-457D-A479-B36EA5FEEF9C&lang=en

**Comment by Dirgham2015** [comment]
I want rhea rings

**Reply by TheLostProbe** [reply]
then download the addon bro tf are you waiting for 😭
